### PR TITLE
Add Modal + TS to account page

### PR DESCRIPTION
**Pull request by amy-corson-ibigroup**
**Description:**

Adds the `PopupWrapper` to the account settings page so users can submit feedback from account settings. 

Also adds TS to the file.

**PR Checklist:**
- [ ] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [ ] Are all languages supported (Internationalization/Localization)?
- [ ] Are appropriate Typescript types implemented?

### CODE REVIEW

File: lib/components/user/account-page.tsx [hunk 1]
```diff
@@ -50,17 +73,20 @@ class AccountPage extends Component {
 
 // connect to the redux store
 
-const mapStateToProps = (state, ownProps) => {
+const mapStateToProps = (state: AppReduxState, ownProps: any) => {
```
Comment (binh-dam-ibigroup) [inline lib/components/user/account-page.tsx:76]: Delete `ownProps`.

File: lib/components/user/account-page.tsx [hunk 2]
```diff
@@ -34,14 +49,22 @@ class AccountPage extends Component {
...
   }
 
-  render () {
-    const {children, subnav = true} = this.props
+  render() {
+    const { children, popupContent, subnav = true } = this.props
     return (
-      <AppFrame SubNav={subnav && SubNav}>
+      // @ts-expect-error TODO: add typing for SubNav
+      <AppFrame SubNav={SubNav}>
```
Comment (binh-dam-ibigroup) [inline lib/components/user/account-page.tsx:65]: The `subNav` boolean prop is still needed. It is used to hide the "My account" header and the "Trips/Settings" tabs when saving a trip.

**Review by binh-dam-ibigroup — CHANGES_REQUESTED** [review]
Use the `subNav` boolean prop and fix the type error on `setPopupContent`, please.

**Review by binh-dam-ibigroup — APPROVED** [review]
Works well, thanks for the changes.

**Review by miles-grant-ibigroup — APPROVED** [review]
Working Great!

**Pull request merged**